### PR TITLE
updating create and debug for VS2017 changes

### PR DESCRIPTION
**Pull request by davidchesnut**
Made the following changes:
- Updated steps to follow VS2017 UI.
- Updated descriptions of VS2017 add-in templates for 2017 changes.
- Removed steps to attach to browser process (this is no longer necessary when making code changes in VS)
- Removed references to MS Project add-ins (these are not possible from VS)

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-vs2017/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810012150062156-431%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9rgO8xFGKl8vtP67xo1ynPQjoMjznLhIiqfquQ7Ce0Y%3D&st=2018-10-01T21%3A46%3A25Z&se=2018-11-01T21%3A51%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/create-and-debug-office-add-ins-in-visual-studio.md [hunk 1]
```diff
@@ -171,57 +144,31 @@ Visual Studio doesn't show validation errors in the  **OUTPUT** window when you
...
-
-### To show an Office Add-in in Project
-
-
-1. In Project, on the  **Project** tab, choose **Office Add-ins**.
-    
-2. In the list that appears, choose your add-in.
-    
-In Visual Studio, you can then set break-points. Then, as you interact with your add-in and step through the code in your HTML, JavaScript, and C# or VB code files.
+1. In Excel or Word, on the  **Insert** tab, choose the **My Add-ins** drop down list. Choose the list from the drop-down arrow, not the button itself which opens the **Office Add-ins** dialog.
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/develop/create-and-debug-office-add-ins-in-visual-studio.md]: Never mind this. The UI I'm seeing may not be public for quite some time.

**Comment by kbrandl** [comment]
@davidchesnut now that Ricky has approved this PR, please **Squash and merge** it into **master** if the content is ready to be published. (Then after you've merged the PR, you can delete the **dc-vs-2017** branch.) Thanks!

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/create-and-debug-office-add-ins-in-visual-studio.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-vs2017/docs/develop/create-and-debug-office-add-ins-in-visual-studio.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810111727180557-431%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=WvXy2SoZKqtHNTO%2BokASipX5SeIPSMX0Q0WRk43GklQ%3D&st=2018-10-11T17%3A23%3A35Z&se=2018-11-11T17%3A28%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**